examples/db-templates/mongodb-ephemeral-template.json

```diff
@@ -152,13 +155,21 @@
   ],
   "parameters": [
     {
+      "name": "MEMORY_LIMIT",
+      "displayName": "Memory limit",
+      "description": "Maximum amount of memory a pod can use",
```

Comment:
technically these are all container limits, not pod limits.  and it should probably be "the" (referring to the container defined in this template) and not "a" (which seems more vague as to what pod/container it's referring to)